packages/mermaid/src/manager.ts

```diff
@@ -82,6 +84,11 @@ export class MermaidManager implements IMermaidManager {
   async renderSvg(text: string): Promise<IMermaidManager.IRenderInfo> {
     const _mermaid = await this.getMermaid();
 
+    // could trigger a false-positive for diagrams about real cervidae, but...
+    if (text.includes('elk')) {
```

Comment:
:rofl: also cities of [Ełk](https://en.wikipedia.org/wiki/E%C5%82k) and [Elk](https://en.wikipedia.org/wiki/Elk_City,_Oklahoma) if wrongly capitalised. And any word containing `elk`? 

- What is the cost of a false positive here?
- Is there a delimiter that could be included in the query?

Reply:
So, `IMermaidManager` is already a token: since this behavior is split up now, we could add a `registerLayoutLoader` which would isolate this further, and allow folks to disable it. Will do an API sketch _after_ it's tested.

The biggest win here would be if there existed an accessible renderer that generated useful HTML representations... any flowchart-kinda thing can be represented as two linked tables, a mindmap as a bulleted list, etc. Way out of scope for this, but might be germaine to your non-human interpretations as well.

As per usual, this would _not_ be portable to other platforms (e.g. Git{Hu,La}b), but they don't (and might never) support elk. Nothing we can do about that, but we _could_ ensure jupyter implementations worked consistently.

Reply:
(also, error sniffing does not work, it doesn't complain at all and just falls back to dagre)